src/components/side-panel/interaction-panel/tooltip-config/tooltip-chicklet.js

```diff
@@ -0,0 +1,170 @@
+// Copyright (c) 2020 7ParkData.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+import React, {Component} from 'react';
+import styled from 'styled-components';
+import {ChickletButton, ChickletTag} from 'components/common/item-selector/chickleted-input';
+import {Circle, Delete} from 'components/common/icons';
+import {TOOLTIP_FORMATS} from 'constants/tooltip';
+import DropdownList from 'components/common/item-selector/dropdown-list';
+import {Tooltip} from 'components/common/styled-components';
+import {FormattedMessage} from 'react-intl';
+import onClickOutside from 'react-onclickoutside';
+
+function getFormatLabels() {
+  return Object.values(TOOLTIP_FORMATS).map(v => v.label);
```

Comment:
you should filter formats based on field types. only show relevant format options

Reply:
We were talking about time formatting which will be in next iteration. Can't we leave it as it is for now?
It's taking to long to implement rn.